operator/src/test/java/oracle/kubernetes/operator/DomainStatusUpdateTestBase.java

```diff
@@ -292,6 +292,40 @@ void whenServerIntentionallyNotStarted_reportItsStateAsShutdown() {
           hasStatusForServer("server1").withState(SHUTDOWN_STATE).withDesiredState(SHUTDOWN_STATE));
   }
 
+  @Test
```

Comment:
I don't understand the purpose of this test. If a server is _intentionally_ not started, that would not be a failure. Reading the JIRA, I think what you want is to test that the server did not come up within some predefined time, but that's not what you're checking, here - and it's not what this test (or the next) claim to be testing.

Reply:
See the scenario in https://jira.oraclecorp.com/jira/browse/OWLS-94702 or ItDiagnosticsFailedCondition#testMSBootFailureStatus.

Reply:
The JIRA specifies that it should be a failure if they don't start within some (presumably configurable) time. There is nothing time-based in either the tests or the implementation. It is perfectly reasonable for the servers not to have started by the time we next update the status; that should not be considered a failure. At the very least, we need to verify that any started servers which are not yet ready have had enough time.